microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md

```diff
@@ -117,3 +117,4 @@ To learn more about the **Source Maps Monitor** tool and how it can help monitor
 * [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](publish-source-maps-to-azure.md)
 * [Map the processed code to your original source code, for debugging](source-maps.md)
 * [Source Maps Monitor tool](../source-maps-monitor/source-maps-monitor-tool.md)
+* [Improvements for connecting DevTools to Azure Artifacts symbol server](../whats-new/2023/07/devtools-115.md#improvements-for-connecting-devtools-to-azure-artifacts-symbol-server) in _What's New in DevTools (Microsoft Edge 115)_.
```

Comment:
instance 1
@captainbrosset - maybe update article per What's New section, since PAT's no longer the only way

Reply:
entered work item for that